src/components/ChatBar.tsx

```diff
@@ -678,6 +679,11 @@ const ChatBar: React.FC = () => {
           <Button onClick={() => handleChangeHeight(-50)}>
             <Typography sx={{ fontWeight: "bold", fontSize: 14 }}>-</Typography>
           </Button>
+          <Button onClick={() => {
+            chrome.runtime.openOptionsPage()
+          }}>
+           <SettingsIcon sx={{fontWeight: "bold", fontSize: 14}}/> 
```

Comment:
Can you increase `fontSize` to `20`? This will make the icon the same size as the rest of the icons.

<img width="548" alt="Screenshot 2024-04-06 at 6 21 17 PM" src="https://github.com/andrewnguonly/Lumos/assets/7654246/147257fa-49d9-482d-9059-6331b45c49ec">